docs/howto/partitions.rst

```diff
@@ -120,28 +146,28 @@ will not be affected for the other partitions.)
 See also
 ^^^^^^^^
 
- * Craft parts: part properties: `organize`_
- * Craft parts: filesets: `specifying paths`_
+* Craft parts: part properties: `organize`_
+* Craft parts: filesets: `specifying paths`_
 
 In environment variables
 ------------------------
 
-You might use these variables in a lifecycle override section of a
-configuration yaml.  For instance:
+You might use these variables in a lifecycle override section of a project
+file.  For instance:
 
 .. code-block:: yaml
 
   override-prime: |
-    cp -R $CRAFT_KERNEL_STAGE/vmlinux $CRAFT_KERNEL_PRIME/
+    cp -R /vmlinux $CRAFT_KERNEL_PRIME/
```

Comment:
I don't know what the original issue was but this is incorrect: this will copy `vmlinux` from the root of the build system (so it'll probably fail)

Reply:
Ok, I didn't understand @cmatsuoka's comment that resulted in this change, I'll ask him to clarify.

Reply:
This should be a relative path